react/NewAddressForm.tsx

```diff
@@ -32,8 +32,8 @@ const NewAddressForm: React.FC<Props> = ({ onAddressCreated }) => {
       <div className="pv3">
         <DeviceCoordinates onSuccess={onAddressCreated} />
       </div>
-      <div className="mt6 w-100 mw6">
-        <LocationCountry />
+      <div className="w-100 mw6">
+        <LocationCountry className="mt6" />
       </div>
```

Comment:
How about the following? I think this change would make no visual difference, but it would eliminate the empty `div` from the DOM.
```suggestion
      <LocationCountry className="w-100 mw6 mt6" />
```

Reply:
:satisfaction:

Reply:
Nice! Done.